config/install/field.field.node.stanford_publication.su_publication_topics.yml

```diff
@@ -0,0 +1,28 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.su_publication_topics
+    - node.type.stanford_publication
+    - taxonomy.vocabulary.stanford_publication_topics
+id: node.stanford_publication.su_publication_topics
+field_name: su_publication_topics
+entity_type: node
+bundle: stanford_publication
+label: Topics
+description: ''
+required: false
```

Comment:
This field is not required yet it is part of the pathautho settings.

Reply:
Pathauto settings ignores that part of the path. so it only adds it when the field is populated. pretty handy

Reply:
What happens when someone adds a tag? Does the URL change? Does a redirect automatically get added?

Reply:
yes. the path changes and a redirect is added.